packages/utils/src/string.ts

```diff
@@ -2,14 +2,14 @@ import * as cosmjsEncoding from "@cosmjs/encoding";
 import * as viem from "viem";
 
 /** Trucates a string with ellipsis, default breakpoint: `num = 8`. */
-export function truncateString(str: string, num = 8) {
+export function truncate(str: string, num = 8) {
   if (str.length <= num) {
     return str;
   }
   return str.slice(0, num) + "...";
 }
 
-export function getShortAddress(
+export function shorten(
```

Comment:
Thoughts on renamin this to  `shortenAddress` since. the latter doesn't convey what is being shortened?

Reply:
I made it more generic since it can be used for any string, like how it's used in asset info page to allow user to copy IBC hash (or base denom)